### ISSUE TITLE
Establish conventions

### ISSUE THREAD

**Issue by jsignell** [pyviz-topics/examples#56]
Just to capture some conventions that need to be added to the docs: 

If there are multiple notebooks in a project then the default command should be:

```yaml
commands:
  notebooks:
    notebook: .
```

UNLESS there are also panels, in which case it should be:

```yaml
commands:
  dashboard:
    unix: panel serve *_panel.ipynb
    supports_http_options: true
  notebooks:
    notebook: .
```

The dashboard command should be deployed at <project>.pyviz.demo.anaconda.com and the notebooks command at <project>-notebooks.pyviz.demo.anaconda.com

**Comment by jbednar** [comment]
I think this is the same as you propose, but to be sure I understand (and please edit this comment if I don't!):

If a project consists of a single notebook to share as a read-only Jupyter notebook, it should use a command like:

```yaml
commands:
  notebook:
```

The notebook should then be deployed at <project>.pyviz.demo.anaconda.com.

If the project contains multiple notebooks or any dashboards, it should be structured like:

```yaml
commands:
  dashboard:
    unix: panel serve *_panel.ipynb
    supports_http_options: true
  notebooks:
    notebook: .
```

where the `dashboard` section is optional.  The dashboard command should be deployed at <project>.pyviz.demo.anaconda.com and the notebooks command at <project>-notebooks.pyviz.demo.anaconda.com

**Comment by jsignell** [comment]
Included in docs after #74
